### DISCUSSION TITLE
Clips deleate

### DISCUSSION THREAD

**Discussion by FrancescoCardone** [ChristopherKlay/StadiaEnhanced#162] [Ideas & Requests]
I'm wondering if its possible to make a system for delete more (more than one) clip (Video) all in one

**Comment by ChristopherKlay** [comment]
It would likely be possible, but that's actually one of the features Stadia apparently works on as well.
The concept of being able to select and download/delete captures has been showcased along with the explore page idea we recently got.

**Reply by FrancescoCardone** [reply]
Oh ok ok 
I didn't know that Stadia works on as well, but why don't anticipate it like you have done with the search bar?😅
